src/app/admin/page/components/menu-placement/sub-menu/sub-menu.component.ts

```diff
@@ -95,6 +95,7 @@ export class SubMenuComponent implements OnInit {
       });
 
     this.dragulaService.dropModel.subscribe((value) => {
+      this.clearFields();

```

Comment:
Put this line in blow if block.
We should clear fields only the requested (related) bagName is our current component bagName